.gitignore

```diff
@@ -31,3 +31,5 @@ build.xml
 GIT_COMMIT
 VERSION
 Dockerfile.local
+profiles.clj
+/repl/
```

Comment:
These are the changes to allow source files dedicated to the REPL. The `profiles.clj` is an idea that I started experimenting with. I went with an update to `project.clj` instead, but I thought I would leave `profiles.clj` in here as well in case someone wanted to add some project-specific customization that isn't checked into source control.